models/service.go

```diff
@@ -36,6 +36,7 @@ type ServiceDetails struct {
 	Dependencies     map[string][]SourceWorkload `json:"dependencies"`
 	Workloads        WorkloadOverviews           `json:"workloads"`
 	Health           ServiceHealth               `json:"health"`
+	Validations      IstioValidations            `json:"validations,omitempty"`
```

Comment:
Perhaps we can always put the validations, just for consistency with the other models/istio_config.go changes